app/components/filter-by.js

```diff
@@ -0,0 +1,29 @@
+  import Component from '@ember/component';
+  import { inject as service } from '@ember/service';
+  import { computed } from '@ember/object';
+
+  export default Component.extend({
+    store: service(),
+
+    allUsers: computed(function () {
+      return this.get('store').findAll('user', { reload: true });
+    }),
+
+    myTeamMembers: computed('allUsers.isFulfilled', function () {
```

Comment:
No need to use `allUsers.isFulfilled `in the trigger. First, you never used this inside this function. Second, since you used `{ reload: true }` in the allUser computed property. It'll automatically retrigger the property. So just use `allUsers` in the trigger.